### PR TITLE
fix(ThinCard): Fix property usage in ThinCard

### LINKED ISSUE #261: ThinCard props behave opposite of expected
This may just be an issue of opinion, but ThinCard's `folder` and `noBorder` props do not make sense to me.

### folder

This shouldn't default to `true`, since it's a modifier.

ThinCard with a folder tab:
```jsx
<ThinCard folder />
```

(Expected) ThinCard without a folder tab:
```jsx
<ThinCard />
```
Actually still renders ThinCard with a folder.

(Actual) ThinCard without a folder tab:
```jsx
<ThinCard folder={false} />
```
This seems somewhat unusual, and isn't how other components (SUIR) behave.

### noBorder

In general, boolean values shouldn't be "not something".

### PR DESCRIPTION
**Pull request by youngbob**
Update property usage to match conventions.
Remove incorrect use of "key" and undocumented "data" prop.
Use classnames helper correctly.

Fixes #261

BREAKING CHANGE: Consumers of ThinCard will need to update their property usage accordingly.

# problem statement

# solution

# discussion

### CODE REVIEW

**Comment by youngbob** [comment]
Yeah, I'm getting completely different results locally from snapjerk, though. It's failing for me on unrelated components but passing on ThinCard.

File: src/components/ThinCard/ThinCard.jsx [hunk 1]
```diff
@@ -10,24 +10,17 @@ import ThinCardDrawer from './ThinCardDrawer'
...
   return (
     <Flexbox
-      key={data.id}
       {...rest}
       flexDirection='column'
-      className={classnames(`octagon thincard__row ${borderClass}`, className)}
+      className={classnames(
+        'octagon',
+        'thincard__row',
+        { thincard__noborder: !border, thincard__folderpad: border },
```
Comment (cdaringe) [inline src/components/ThinCard/ThinCard.jsx]: man, i gotta read the docs!  this is so much better than my ternaries everywhere

**Comment by cdaringe** [comment]
i'll pull and try.  CI says [a](https://372-77168631-gh.circle-artifacts.com/0/home/circleci/repo/snaps/run/1532636380477-diff/report/a-ThinCard-simple-card-example-chrome.png) and [b](https://372-77168631-gh.circle-artifacts.com/0/home/circleci/repo/snaps/run/1532636380477-diff/report/b-ThinCard-simple-card-example-chrome.png) dont align

**Comment by cdaringe** [comment]
> Yeah, I'm getting completely different results locally from snapjerk, though.

you may need to run a `yarn build` first*.  test should perhaps build first

**Comment by wa11-e** [comment]
Woohoo! new surge deployment available for viewing! :tada: [surgereviewiraoedpxrsvezil.surge.sh](http://surgereviewiraoedpxrsvezil.surge.sh)

**Pull request merged**